client/shipping-label/components/label-purchase-modal/style.scss

```diff
@@ -0,0 +1,94 @@
+@import './step-confirmation-button/style';
+@import './address-step/style';
+@import './packages-step/style';
+@import './rates-step/style';
+
+
+.label-purchase-modal__sidebar {
+	flex-basis: 100%;
+	padding: 32px;
+	margin-bottom: 61px;
+
+	@include breakpoint( ">660px" ) {
+		flex-basis: 40%;
+		padding: 8px 0px 8px 16px;
+	}
+
+	@include breakpoint( ">960px" ) {
+		flex-basis: 30%;
+	}
+}
+
+.label-purchase-modal__content {
+	display: flex;
+	flex-direction: column;
+	height: 100%;
+
+	.gridicon.is-success {
```

Comment:
We do. We apply the `is-whatever` class to both the `<Gridicon>` and to the step summary. We want `error` summaries to be red, but we don't want them changing color in the other cases (yellow is hard to read, and green isn't needed for the "normal" case of a step that is correct).